akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala

```diff
@@ -21,16 +21,18 @@ class TimeoutsSpec extends AkkaSpec {
 
     "pass through elements unmodified" in assertAllStagesStopped {
       Await.result(
-        Source(1 to 100).via(Timeouts.initalTimeout(2.seconds)).grouped(200).runWith(Sink.head),
+        Source(1 to 100).initialTimeout(2.seconds).grouped(200).runWith(Sink.head),
         3.seconds) should ===(1 to 100)
     }
 
     "pass through error unmodified" in assertAllStagesStopped {
       a[TE] shouldBe thrownBy {
         Await.result(
-          Source(1 to 100).concat(Source.failed(TE("test")))
-            .via(Timeouts.initalTimeout(2.seconds))
-            .grouped(200).runWith(Sink.head),
+          Source(1 to 100).
+            concat(Source.failed(TE("test"))).
+            initialTimeout(2.seconds).
+            grouped(200).
```

Comment:
I addressed your formatting comment :)

On Sun, Nov 1, 2015 at 7:46 PM, Konrad Malawski <notifications@github.com>
wrote:

> In akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala
> <https://github.com/akka/akka/pull/18827#discussion_r43585784>:
>
> >          3.seconds) should ===(1 to 100)
> >      }
> >
> >      "pass through error unmodified" in assertAllStagesStopped {
> >        a[TE] shouldBe thrownBy {
> >          Await.result(
> > -          Source(1 to 100).concat(Source.failed(TE("test")))
> > -            .via(Timeouts.initalTimeout(2.seconds))
> > -            .grouped(200).runWith(Sink.head),
> > +          Source(1 to 100).
> > +            concat(Source.failed(TE("test"))).
> > +            initialTimeout(2.seconds).
> > +            grouped(200).
>
> that one it can't handle currently, would be nice. Do you want to create a
> ticket on their repo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/18827/files#r43585784>.
>



-- 
Cheers,
√